frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1943,7 +1965,8 @@
                     "method_name": vm.method_name,
                     "method_description": vm.method_description,
                     "project_url": vm.project_url,
-                    "publication_url": vm.publication_url
+                    "publication_url": vm.publication_url,
+                    "submission_metadata":JSON.stringify(vm.submissionMetaAttributes[0].attributes)
```

Comment:
@burnerlee this should be `submission_meta_attributes` right? Did you check if the values from UI are getting updated for the submission in DB after submitting?

Reply:
@Ram81 i think it would work using the above code. submission_meta_attributes are located in vm.submissionmetadata and also it does not contain the field for the value of the attribute. So this variable would be used I think. Yes, they are getting updated, the updated values have also been shown in the gif uploaded above for reference